### PR TITLE
Update ROADMAP.md for February 2021

### PR DESCRIPTION
**Pull request by youngnick**
Signed-off-by: Nick Young <ynick@vmware.com>

### CODE REVIEW

**Comment by youngnick** [comment]
As discussed in today's community meeting, here's the updated ROADMAP.md.

Please feel free to comment with suggestions, comments, or fury at my habit of pulling delivery timelines out of nothing :).

I think that we should attempt to update this document after each release, so it doesn't go six months between updates.

**Review by sunjayBhatia — APPROVED** [review]

File: ROADMAP.md [hunk 1]
```diff
@@ -15,22 +15,38 @@ Please open an issue to track any initiative on the roadmap of Contour (Usually
...
 Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Contour.
 
-`Last Updated: August 2020`
+`Last Updated: February 2021`
+
+|Theme|Description|Timeline|Issue|Notes|
+|--|--|--|--|--|
+|Compatibility|Implement Kubernetes Ingress V1 specification (requires v1.16)|March 2021|[projectcontour/contour#2139](https://github.com/projectcontour/contour/issues/2139)||
+|Performance|Rate limiting support|February 2021|[projectcontour/contour#370](https://github.com/projectcontour/contour/issues/370)| Local rate limiting support arrived in 1.12 in January 2021, global rate limiting support is due in 1.13, February 2021.|
```
Comment (skriss) [inline ROADMAP.md:23]: nit: move this line up one for proper date ordering

File: ROADMAP.md [hunk 2]
```diff
@@ -15,22 +15,38 @@ Please open an issue to track any initiative on the roadmap of Contour (Usually
...
+|General|Wildcard Path Matching|March 2021|[projectcontour/contour#2138](https://github.com/projectcontour/contour/issues/2138)|A limited form of this is required for both Ingress and Gateway-APIs.|
+|Observability|Support Access Log Service|April 2021|projectcontour/contour#1291||
```
Comment (skriss) [inline ROADMAP.md:25]: looks like this is actually #1691. Also, haven't heard many requests for this one - does it make sense to be so high on the roadmap?

Comment (danehans) [reply ROADMAP.md:25]: The link for this item is missing too.

File: ROADMAP.md [hunk 3]
```diff
@@ -15,22 +15,38 @@ Please open an issue to track any initiative on the roadmap of Contour (Usually
...
+|Gateway APIs|Introduce support for upstream Gateway APIs|April 2021|[projectcontour/contour#2287](https://github.com/projectcontour/contour/issues/2287)||
```
Comment (danehans) [inline ROADMAP.md:26]: I suggest linking to https://github.com/kubernetes-sigs/gateway-api for context.

File: ROADMAP.md (hunk 2 above)
Comment (youngnick) [reply ROADMAP.md:25]: Ah, this one is one we committed to previously on our roadmap. I'd rather not have things vanish from the roadmap entirely. It seems like a relatively small piece of work, but we can definitely push it back.

I'll fix the link, thanks!

**Review by danehans — APPROVED** [review]

**Review by skriss — APPROVED** [review]

**Pull request merged**